test/test.js

```diff
@@ -183,6 +183,39 @@ describe('vite', () => {
       })
     }
 
+    test('SFC normal css w/ @import', async () => {
+      const el = await page.$('.style-at-import')
+      expect(await getComputedColor(el)).toBe('rgb(255, 0, 0)')
+      if (!isBuild) {
+        await updateFile('css-@import/testCssImportBoundary.css', (content) =>
+          content.replace('red', 'rgb(0, 0, 0)')
+        )
+        await expectByPolling(() => getComputedColor(el), 'rgb(0, 0, 0)')
+      }
+    })
+
+    test('SFC preprocessor w/ @import', async () => {
+      const el = await page.$('.style-at-import-scss')
+      expect(await getComputedColor(el)).toBe('rgb(255, 0, 0)')
+      if (!isBuild) {
+        await updateFile('css-@import/testCssImportBoundary.scss', (content) =>
+          content.replace('red', 'rgb(0, 0, 0)')
+        )
+        await expectByPolling(() => getComputedColor(el), 'rgb(0, 0, 0)')
+      }
+    })
+
+    // test('SFC <style module> w/ @import', async () => {
+    //   const el = await page.$('#css-modules-at-import-sfc')
+    //   expect(await getComputedColor(el)).toBe('rgb(255, 0, 0)')
+    //   if (!isBuild) {
+    //     await updateFile('css-@import/testCssImportBoundary.scss', (content) =>
+    //         content.replace('red', 'rgb(0, 0, 0)')
+    //     )
+    //     await expectByPolling(() => getComputedColor(el), 'rgb(0, 0, 0)')
```

Comment:
This test is ok when run dev when broswer.But it fail in test.I don't know why.So I'm need you take a look. Thanks!